appendix/backup-and-restore/backup.rst

```diff
@@ -0,0 +1,48 @@
+Create Backup
+*************
+
+.. code-block:: sh
+
+   # Zammad backup started - Fri Jan 21 17:53:44 CET 2022!
+
+   creating file backup...
+    ... as full dump
+   creating postgresql backup...
+   Ensuring dump permissions ...
+
+   # Zammad backuped successfully - Fri Jan 21 17:53:57 CET 2022!
+
+*Sample backup process with default*
+:doc:`settings </appendix/backup-and-restore/configuration>`.
+
+--------------------------------------------------------------------------------
+
+.. note::
+
+   🤓 Before running your first backup, please have a look at
+   :doc:`/appendix/backup-and-restore/configuration`.
+
+In general, running a Zammad backup is as simple as running
+
+.. code-block:: sh
+
+   $ /opt/zammad/contrib/backup/zammad_backup.sh
+
+Please make sure to test the backup function manually with the user
+you're planning the backup first. This ensures that your backup really
+is running as expected.
+
+Remarks
+   | The backup script can be either run as ``zammad`` or ``root`` user.
+   | Stopping Zammad is not required technically, but may be in your use case!
```

Comment:
I would like to recommend to stop Zammad during the backup for data consistency reasons.